reference/functions/version_compare.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: version_compare
+published: true
+---
+
+[%CFEngine_function_prototype(string)%]
```

Comment:
```suggestion
[%CFEngine_function_prototype(version1, comparison, version2)%]
```
I think you need to provide the arguments you want it to display.

Reply:
Thanks :)